kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -38,77 +33,25 @@
         return !(Object.keys(this.$slots).length === 0 && this.$slots.constructor === Object);
       },
     },
+    components: {
+      UiButton: require('keen-ui').UiButton,
+    },
   };
 
 </script>
 
 
-<style lang="stylus">
-
-  @require '~kolibri.styles.coreTheme'
-
-  /*
-    WARNING -- these styles are unscoped.
-    ONLY include styles that need to apply to SVGs inserted into the slot.
-    Make sure everything here is scoped under the .icon-button-scope class.
-  */
-
-  .icon-button-scope
-    svg
-      vertical-align: middle
-      fill: $core-action-normal
-      transition: fill $core-time ease-out
-    &:hover svg
-      fill: $core-action-dark
-    &:disabled svg
-      fill: $core-text-disabled
-
-  // styles specific to primary button
-  .icon-button-scope.primary
-    svg
-      fill: $core-bg-canvas
-      transition: fill $core-time ease-out
-    &:hover svg
-      fill: $core-bg-canvas
-    &:disabled svg
-      fill: $core-bg-canvas
-
-</style>
-
-
 <style lang="stylus" scoped>
 
-  @require '~kolibri.styles.coreTheme'
-
-  button
-    padding: 0.2em 2em
-    line-height: inherit
-
-  button.single-line
-    height: 36px
+  @import '~keen-ui/dist/keen-ui.css'
 
-  .primary
-    border: none
-    color: $core-bg-canvas
-    background-color: $core-action-normal
-    transition: background-color $core-time ease-out
-
-    &:hover
-      color: $core-bg-canvas
-      background-color: $core-action-dark
-    &:disabled
-      color: $core-bg-canvas
-      background-color: $core-text-disabled
+</style>
 
-  /* displayed to visually balance an icon */
-  .icon-padding
-    margin-right: 2px
 
-  .btn-text
-    vertical-align: middle
+<style lang="stylus">
 
-  .btn-bottom-text
-    display: block
-    margin-top: 0.4em
+  .ui-button__content svg
```

Comment:
Well, the point is that we "shouldn't" reference the `.ui-button__content` class because it's an implementation detail of keen that we don't have control over.